src/encoded/static/components/status.js

```diff
@@ -0,0 +1,352 @@
+import React from 'react';
```

Comment:
this has nothing to do with the JS code per se - but aren't we duplicating this login in python @keenangraham ?  I feel like this should be configure in the schemas or some other shared config file.